modules/nw-cluster-mtu-change.adoc

```diff
@@ -0,0 +1,482 @@
+
+... To find the primary network interface, enter the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name> -- chroot /host ip route list match 0.0.0.0/0 | awk '{print $5 }'
+----
++
+--
+where:
+
+`<node_name>`:: Specifies the name of a node in your cluster.
+--
+
+... To find the connection profile that NetworkManager created for the interface name returned from the previous command, enter the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name> -- chroot /host nmcli c | grep <interface>
+----
++
+--
+where:
+
+`<interface>`:: Specifies the name of the primary network interface.
+--
++
+.Example output
+[source,text]
+----
+Wired connection 1  46da4a6a-xxxx-xxxx-xxxx-ac0ca900f213  ethernet  ens3
+----
+
+... To get the file name of the NetworkManager connection configuration for the primary network interface, enter the following command:
++
+[source,terminal]
+----
+oc debug node/<node_name> -- chroot /host nmcli -g UUID,FILENAME c show | grep <uuid> | cut -d: -f2
+----
++
+--
+where:
+
+`<node_name>`:: Specifies the name of a node in your cluster.
+`<uuid>`:: Specifies the UUID of the NetworkManager connection profile.
+--
++
+.Example output
+[source,text]
+----
+/run/NetworkManager/system-connections/Wired connection 1.nmconnection
+----
+
+... To copy the NetworkManager connection configuration from the node to make changes, enter the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name> -- chroot /host cat "<profile_path>" > config.nmconnection
+----
++
+--
+where:
+
+`<node_name>`:: Specifies the hardware MTU for the primary network interface.
+`<profile_path>`:: Specifies the file system path of the NetworkManager connection from the previous step.
+--
++
+.Example NetworkManager connection configuration
+[source,ini]
+----
+[connection]
+id=Wired connection 1
+uuid=3e96a02b-88e0-3707-ad5d-61db28678130
+type=ethernet
+autoconnect-priority=-999
+interface-name=enp1s0
+permissions=
+timestamp=1644109633
+
+[ethernet]
+mac-address-blacklist=
+
+[ipv4]
+dns-search=
+method=auto
+
+[ipv6]
+addr-gen-mode=stable-privacy
+dns-search=
+method=auto
+
+[proxy]
+
+[.nmmeta]
+nm-generated=true
+----
+
+... Edit the NetworkManager configuration file saved in the `config.nmconnection` file from the previous step:
++
+--
+**** Set the following values:
+***** `802-3-ethernet.mtu`: Specify the MTU for the primary network interface of the system.
+***** `connection.interface-name`: Optional: Specify the network interface name that this configuration applies to.
+***** `connection.autoconnect-priority`: Optional: Specify an integer priority value greater than any other ConnectionManager connection profiles to ensure this profile is used.
```

Comment:
I would rephrase to `Consider specifying an integer priority value above 0 to ensure this profile is used over other profiles for the same interface`. For OVN-K, priority should not be set at or above 100.